src/rpc/net.cpp

```diff
@@ -446,6 +451,10 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
             "  \"subversion\": \"/Satoshi:x.x.x/\",     (string) the server subversion string\n"
             "  \"protocolversion\": xxxxx,              (numeric) the protocol version\n"
             "  \"localservices\": \"xxxxxxxxxxxxxxxx\", (string) the services we offer to the network\n"
+            "  \"localservicesnames\": [              (array) the services we offer to the network, in human-readable form\n"
+            "      \"SERVICE_NAME\",         (string) the service name if it is recognised\n"
```

Comment:
If you need to change this for other reasons, might as well remove the "if it is recognised". If we offer a service, we better recognize it.
```suggestion
            "      \"SERVICE_NAME\",         (string) the service name\n"
```

Reply:
That the change you requested above ...

Reply:
Fixed